examples/05-Q3D/Q2D_Armoured_Cable.py

```diff
@@ -0,0 +1,392 @@
+arm_strand_1_id.duplicate_around_axis('Z', '360deg/n_arm_strands', nclones='n_arm_strands')
+arm_strand_names = mod2D.get_objects_w_string('arm_strand')
+
+#####################################################################################
+# Create region
+# ~~~~~~~~~~~~~
+
+oeditor = Q2D_DS.odesign.SetActiveEditor("3D Modeler")
+oeditor.CreateRegion(
+    [
+        "NAME:RegionParameters",
+        "+XPaddingType:=", "Percentage Offset",
+        "+XPadding:=", "500",
+        "-XPaddingType:=", "Percentage Offset",
+        "-XPadding:=", "500",
+        "+YPaddingType:=", "Percentage Offset",
+        "+YPadding:=", "500",
+        "-YPaddingType:=", "Percentage Offset",
+        "-YPadding:=", "500",
+        "+ZPaddingType:=", "Percentage Offset",
+        "+ZPadding:=", "0",
+        "-ZPaddingType:=", "Percentage Offset",
+        "-ZPadding:="	, "0",
+		[
+			"NAME:BoxForVirtualObjects",
+			[
+				"NAME:LowPoint",
+				1,
+				1,
+				1
+			],
+			[
+				"NAME:HighPoint",
+				-1,
+				-1,
+				-1
+			]
+		]
+	],
+	[
+		"NAME:Attributes",
+		"Name:="		, "Region",
+		"Flags:="		, "Wireframe#",
+		"Color:="		, "(143 175 143)",
+		"Transparency:="	, 0,
+		"PartCoordinateSystem:=", "Global",
+		"UDMId:="		, "",
+		"MaterialValue:="	, "\"vacuum\"",
+		"SurfaceMaterialValue:=", "\"\"",
+		"SolveInside:="		, False,
+		"ShellElement:="	, False,
+		"ShellElementThickness:=", "nan ",
+		"ReferenceTemperature:=", "nan ",
+		"IsMaterialEditable:="	, True,
+		"UseMaterialAppearance:=", False,
+		"IsLightweight:="	, False
+	])
+
+##########################################################
+# Assign conductors and reference ground
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+obj = [Q2D_DS.modeler.get_object_from_name(i) for i in cond_names]
+[Q2D_DS.assign_single_conductor(name ='C1 ' +str(obj.index(i) + 1), target_objects=i, conductor_type='SignalLine') for i
+ in obj]
+obj = [Q2D_DS.modeler.get_object_from_name(i) for i in arm_strand_names]
+Q2D_DS.assign_single_conductor(name="gnd", target_objects=obj, conductor_type="ReferenceGround")
+mod2D.fit_all()
+
+##########################################################
+# Assign design settings
+# ~~~~~~~~~~~~~~~~~~~~~~
+
+lumped_length = "100m"
+q2d_des_settings = Q2D_DS.design_settings()
+q2d_des_settings['LumpedLength'] = lumped_length
+Q2D_DS.change_design_settings(q2d_des_settings)
+
+##########################################################
+# Insert setup and frequency sweep
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+q2d_setup = Q2D_DS.create_setup(setupname=sName)
+q2d_sweep = q2d_setup.add_sweep(sweepname=swName)
+q2d_sweep.props["RangeType"] = "LogScale"
+q2d_sweep.props["RangeStart"] = "0Hz"
+q2d_sweep.props["RangeEnd"] = "3MHz"
+q2d_sweep.props["RangeCount"] = 10
+q2d_sweep.props["RangeSamples"] = 1
+q2d_sweep.update()
+
+##########################################################
+# Analyze setup
+# ~~~~~~~~~~~~~
+
+Q2D_DS.analyze(setup_name=sName, num_cores=4, num_tasks=2)
```

Comment:
@tizianrot is there a way to reduce computational time in analysis? it takes too much at the moment